GitOut/Features/Git/Stage/GitStagePage.xaml

```diff
@@ -0,0 +1,263 @@
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <ScrollViewer
+                    x:Name="DocumentScroll"
+                    VerticalScrollBarVisibility="Auto"
+                    HorizontalScrollBarVisibility="Auto"
+                >
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto" MinWidth="20"/>
+                            <ColumnDefinition Width="*"/>
+                        </Grid.ColumnDefinitions>
+                        <ItemsControl ItemsSource="{Binding SelectedDiff.LineNumbers}">
+                            <ItemsControl.ItemTemplate>
+                                <DataTemplate DataType="{x:Type local:LineNumberViewModel}">
+                                    <Grid
+                                        Margin="4 0"
+                                        TextElement.FontFamily="Consolas sans-serif"
+                                        TextElement.FontSize="12"
+                                        TextElement.Foreground="{DynamicResource MaterialGray400}"
+                                        HorizontalAlignment="Stretch"
+                                    >
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="*"/>
+                                            <ColumnDefinition Width="*"/>
+                                        </Grid.ColumnDefinitions>
+                                        <Grid HorizontalAlignment="Stretch">
+                                            <TextBlock
+                                                TextAlignment="Right"
+                                                Text="{Binding WorkingTreeLineNumber, TargetNullValue=' '}"
+                                            />
+                                        </Grid>
+                                        <TextBlock
+                                            Grid.Column="1"
+                                            Margin="3 0 0 0"
+                                            TextAlignment="Right"
+                                            Text="{Binding IndexLineNumber}"
+                                        />
+                                    </Grid>
+                                </DataTemplate>
+                            </ItemsControl.ItemTemplate>
+                        </ItemsControl>
+                        <FlowDocumentScrollViewer
+                            Grid.Column="1"
+                            Document="{Binding SelectedDiff.Document}"
+                            HorizontalAlignment="Stretch"
+                            VerticalScrollBarVisibility="Disabled"
+                            HorizontalScrollBarVisibility="Disabled"
+                            PreviewMouseWheel="TunnelEventToParentScroll"
```

Comment:
Yes, but maybe not at this moment, but rather when we need it.
I could see where this would be handy in e.g. a history/blame window, so I'm not against the idea. :thinking: